### PR TITLE
Travis is done and never coming back

### PR DESCRIPTION
**Pull request by chrisseaton**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -2,14 +2,7 @@
   require 'simplecov'
   require 'coveralls'
 
-  if ENV['TRAVIS']
```
Comment (eregon) [inline spec/spec_helper.rb:5]: Should it be `ENV['CI']` instead?

**Review by eregon — APPROVED** [review]

File: spec/spec_helper.rb (hunk 1 above)
Comment (chrisseaton) [reply spec/spec_helper.rb:5]: The branch hasn't been run the whole time we've been using GitHub Actions. If we want a different coverage output format in the future we can pick it then.

File: spec/concurrent/actor_spec.rb [hunk 2]
```diff
@@ -148,7 +148,7 @@ def on_message(message)
           end
         end
 
-        it 'terminates with all its children', notravis: true do
```
Comment (eregon) [inline spec/concurrent/actor_spec.rb:151]: I wonder if this tag was somehow avoiding running transient tests in CI.
Maybe the failure in https://github.com/ruby-concurrency/concurrent-ruby/runs/5553488676?check_suite_focus=true is related (but maybe not)

Comment (eregon) [reply spec/concurrent/actor_spec.rb:151]: Given the Rakefile above it sounds like they were excluded, even on GitHub Actions before this PR

**Pull request merged**